apps/synapse-portal-framework/src/components/challenges/NewChallengesSection/NewChallengesSection.tsx

```diff
@@ -37,16 +34,20 @@ const NewChallengesSection = ({
 
   const dataRows = queryResultBundle?.queryResult?.queryResults.rows ?? []
 
+  const filteredDataRows =
```

Comment:
Looks like the second instance of this type of method.  Maybe refactor to a util, and have it take a param for the filter value (such as 'new')?